lessons/Data-Science/ai-decision-trees/langs/en-us/pages/level-2.adoc

```diff
@@ -0,0 +1,60 @@
+= Decision Tree: Level 2
+
+[.linkInstructions]#Use the @handout{decision-tree-data.adoc, "training data"} to respond to the prompts below.# Let's decide what decision nodes to use in the 2nd level of our tree.
+
+== Shopping History as the Decision Node
+
+@n Using information from the "age" column and the "shopping history" column, fill in the blanks below.
+
+@indented{a) The training data indicates that *frequent customers in their teens* @fitbruby{10em}{@ifsoln{will}}{will / will not} buy the game.
+
+b) The training data indicates that *new customers in their teens* probably @fitbruby{10em}{@ifsoln{will}}{will / will not} buy the game.
+
+@vspace{1ex}
```

Comment:
Why is this space here? The uneven spacing between the lines on the page is a little jarring